prepare_ui_tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+### Download Docker Images
+echo "you must have permission to run docker."
```

Comment:
I'd suggest to either have a script check wether the user has a permission to run docker, or to echo it as a `Warning: ...`